packages/mesos-modules/buildinfo.json

```diff
@@ -3,7 +3,7 @@
     "single_source" : {
       "kind": "git",
       "git": "https://github.com/dcos/dcos-mesos-modules.git",
-      "ref": "8d09c2d44f606df23a57254ebcbc5f2722a3b677",
-      "ref_origin": "master"
+      "ref": "86f14f9402e324f78385a0b7b9509079dda7c0ee",
+      "ref_origin": "1.8.x"
```

Comment:
I can't find the systemd header reference in configure.ac anymore. I'm not sure what I was looking at, but it's gone now.